src/Uno.Sdk/packages.json

```diff
@@ -121,7 +121,7 @@
   },
   {
     "group": "WinAppSdkBuildTools",
-    "version": "10.0.26100.1742",
+    "version": "10.0.22621.3233", // Keep this version until https://github.com/microsoft/WindowsAppSDK/issues/4480 is resolved
```

Comment:
It's likely that the automated tooling will force the upgrade to the latest version.  

We probably should be excluding this group here:

https://github.com/unoplatform/uno.templates/blob/6974d53309e8bb2de1ce13d6f23c0d5149d10020/tools/Uno.Sdk.Updater/Program.cs#L294-L298

Reply:
Done